dune_api_scripts/store/solver_list.py

```diff
@@ -0,0 +1,62 @@
+"""Modifies and executed dune query for today's data"""
+from __future__ import annotations
+
+import dataclasses
+import json
+import os
+from dataclasses import dataclass
```

Comment:
Nitpick: It might be that I've been away from Python for a long time, because it does look weird these two `dataclasses` imports. Is that ok?

Reply:
Yea that is a bit weird. I will have to take a look at what happened here. I generally let my IDE to the imports for me. One of these is likely a mistake.